pkg/machinery/config/types/v1alpha1/v1alpha1_types.go

```diff
@@ -972,7 +972,9 @@ type InstallDiskSelector struct {
 	//     - nvme
 	//     - sd
 	Type InstallDiskType `yaml:"type,omitempty"`
-	//   description: Disk bus path.
+	//   description: |
+	//	    Disk bus path.
```

Comment:
using mixed indentation here (`tab` + `space`), this causes errors parsing this value

Reply:
Thanks - I'll  fix when I'm back online in a few hours